package.json

```diff
@@ -5,45 +5,7 @@
   "useWorkspaces": true,
   "workspaces": [
     "golang/cosmos",
-    "packages/assert",
-    "packages/import-manager",
-    "packages/sparse-ints",
-    "packages/store",
-    "packages/bundle-source",
-    "packages/import-bundle",
-    "packages/eventual-send",
-    "packages/governance",
-    "packages/promise-kit",
-    "packages/tame-metering",
-    "packages/transform-metering",
-    "packages/same-structure",
-    "packages/stat-logger",
-    "packages/swing-store",
-    "packages/SwingSet",
-    "packages/swingset-runner",
-    "packages/telemetry",
-    "packages/ERTP",
```

Comment:
because the package was renamed to lowercase, I was getting this:
<img width="845" alt="Screen Shot 2022-02-15 at 7 00 18 AM" src="https://user-images.githubusercontent.com/21505/154103198-67e46b09-5cfe-4c3e-8fb3-c075c2a5b390.png">
 
 Took me a while to figure out.